package.json

```diff
@@ -34,24 +30,23 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "audit-ci": "^4.0.0",
-    "chokidar-cli": "^2.1.0",
-    "concurrently": "^6.0.0",
-    "husky": "=4",
-    "lint-staged": "^11.2.6",
-    "prettier": "^2.4.1",
+    "chokidar-cli": "^3.0.0",
+    "concurrently": "^7.6.0",
+    "husky": "^8.0.2",
+    "lint-staged": "^12.5.0",
+    "prettier": "2.8.0",
     "rimraf": "^3.0.2",
-    "turbo": "~1.3.2"
+    "turbo": "~1.6.3"
   },
   "resolutions": {
-    "hosted-git-info": ">=3.0.8",
-    "jimp": ">=0.16.1",
-    "trim-newlines": "^3.0.1",
-    "minimatch": "^3.0.5",
-    "glob-parent": "^6.0.1",
-    "jpeg-js": "^0.4.3",
-    "ansi-regex": "^5.0.1",
-    "node-fetch": "^2.6.7",
-    "async": "^3.2.2"
+    "hosted-git-info": "6.1.1",
+    "jimp": "0.16.2",
+    "trim-newlines": "3.0.1",
```

Comment:
`trim-newlines` latest version is `4.0.2`. Should we use that instead? Any reason why we are sticking with the `3.0.1` tag and not `latest`? ([source](https://www.npmjs.com/package/trim-newlines?activeTab=versions))

![image](https://user-images.githubusercontent.com/45748283/207967066-748ddaad-56f8-4b30-b84e-40e4c66b6d38.png)

Reply:
i tried to keep all resolutions to the same major version. most of these resolutions were done solely to pass `yarn audit`, so there is no reason to upgrade them to latest and risk breaking changes